src/config.js

```diff
@@ -4,7 +4,7 @@ export const isProd: boolean = process.env.NODE_ENV === 'production';
 export const LOGKEEPER_BASE: string =
   process.env.REACT_APP_LOGKEEPER_BASE || 'https://logkeeper.mongodb.org';
 export const EVERGREEN_BASE: string =
-  process.env.REACT_APP_EVERGREEN_BASE || 'https://evergreen.mongodb.com';
+  process.env.REACT_APP_EVERGREEN_BASE || '';
```

Comment:
What is the difference here? It seems like using `http://evergreen.mongodb.com` is a fine default. Or am I not understanding this code correctly?

Reply:
It has to do with how domain names are resolved in the browser. 
Browsers will try to apply the origin domain name if one is not supplied. 
Before when lobster would try to fetch logs it would always hit `https://evergreen.mongodb.com/test_log/....` 
Since `https://evergreen.mongodb.com` was always hard coded in.

When lobster is served on staging it would always hit the hardcoded value instead of hitting `https://evergreen-staging.corp.mongodb.com/test_log/....` it would hit the value from above.

By omitting the hard coded domain name the request will always go to `/test_log/...` and by default the request will be directed to the same domain name the page is served from.

So lobster pages loaded on https://evergreen.mongodb.com will always try to hit production evergreen
and lobster pages loaded on https://evergreen-staging.corp.mongodb.com will always try to hit staging evergreen.

Reply:
Thank you!